app/controllers/curriculum-inventory-reports.js

```diff
@@ -128,22 +128,24 @@ export default Controller.extend({
     });
   }),
 
-  reports: computed('selectedSchool', 'selectedProgram', function(){
+  reports: computed('selectedProgram', function(){
     let defer = RSVP.defer();
     const selectedSchool = this.get('selectedSchool');
```

Comment:
selectedSchool is getting double called now.  If it is a promise this should be removed otherwise the next line should be removed.